### DISCUSSION TITLE
Multiple W5500 approach

### DISCUSSION THREAD

**Discussion by f3rn4ndes** [sstaub/Ethernet3#57] [Q&A]
Dear Sirs

I am needing connect 2 W5500 using the ESP32 module. First approach : sharing the SPI and individuals pins to CS, INT and RESET of W5500 modules. Second approach: using a complete SPI to each module, separately. The question regarding this library: Does it support manage 2 modules ? Thanks In Advance.

**Comment by sstaub** [comment]
I don't think this impossible in anyway. The library, also the original, can only handle one WIZnet chip.

**Comment by proasnet** [comment]
dear @sstaub , I am interesting about two W5500 too.
I need make a gateway between PLC Siemens S7-1200 in the device network, and external port with more services as modbus TCP, web and MQTT to another network.
Is possible with your library use more than one ethernet shield?
Thank you.

**Comment by sstaub** [comment]
No.
